### DISCUSSION TITLE
Sugestões de sites para clonagem para fins educativos.

### DISCUSSION THREAD

**Discussion by Mikabers** [frontendbr/forum#2038] [Dicas]
**Olá gente.** 

Eu queria me desafiar e aprimorar um pouco minhas habilidades em **HTML, CSS e JS**, e queria sugestões de sites legais pra clonar, sejam sites mais fáceis de fazer ou sites um pouco complexos. 

Agradeço se alguém responder e obrigado pela atenção! 😊

**Comment by vczb** [comment]
A home page do Github :)

**Reply by Mikabers** [reply]
Se eu te dizer que eu tava pensando no Github antes de ver sua mensagem. 😅 

Eu tava indo estudar o site, pra fazer aqui, muito obrigado pela sugestão. 😊

**Comment by wilsonneto-swe** [comment]
Pega um site/app que vc use nbo dia a dia e que irá se divertir clonando, minha sugestão.

**Reply by Mikabers** [reply]
Irei sim, tou vendo aqui nesse exato momento, obrigado pela sugestão. 😊

**Comment by ninetails** [comment]
Ou pegue um site que você nunca viu na vida:

> https://dribbble.com/shots/popular/web-design

E monte responsivo utilizando um deles como inspiração rs

Ou faz um [challenge](https://github.com/felipefialho/frontend-challenges)

**Comment by theandersonn** [comment]
Duas iniciativas legais com desafios.
- https://www.devchallenge.com.br/
- https://www.frontendmentor.io/challenges

**Comment by oandresantoss** [comment]
Já entrei nessa também!
